frontend/public/components/_terminal.scss

```diff
@@ -12,3 +12,14 @@
   padding: 0;
   background-color: $color-pf-black;
 }
+
+.console-collapse-link {
+  color: $color-pf-blue-400;
+  position: fixed;
+  right: 18px;
+  top: 4px;
+  z-index: 256;  // to get above 'terminal xterm fullscreen's z-index of 255
+  .fa {
+    font-weight: 700;
```

Comment:
Not sure about the font-weight change. I think would be more effective to leave font-weight and give it a color that contrasts more. This probably doesn't meet accessibility requirements for contrast